.github/workflows/e2e-test-windows.yaml

```
@@ -0,0 +1,174 @@
+        - windows-version: '10'
+          windows-featurepack: '23h2-ent'
+        - windows-version: '11'
+          windows-featurepack: '22h2-ent-g2'
+
+    steps:
+    - name: Download qe oci image
+      id: download-qe-oci-image-artifact
+      uses: actions/download-artifact@v7
+      with:
+        name: macadam-e2e-windows-amd64
+        run-id: ${{inputs.trigger-workflow-run-id}}
```

actionlint (1.7.10)

[error] property "trigger-workflow-run-id" is not defined in object type {}

(expression)